pages/Category.vue

```diff
@@ -46,26 +51,26 @@
             </SfSelectOption>
           </SfSelect>
           <SfButton
-            class="sf-button--text navbar__filters-button sort-by__button mobile-only"
+            class="sf-button--text navbar__button mobile-only"
             @click="$refs.SortBy.toggle()"
           >
             {{ $t('Sort By') }}
             <ASortIcon />
           </SfButton>
         </div>
+
```

Comment:
I think we can add a comment here that if a user wants to add additional items to the nabber they should insert them right here, before the counter.

Reply:
Ok I will do that